api/swagger/swagger.yaml

```diff
@@ -72,6 +72,15 @@
           - "VIPER Rating"
         produces:
           - "application/json"
+    /ingest:
```

Comment:
i think we should leave this out of the swagger for now, we probably don't want external clients to call it, and we can always go around the back of the gateway